runtime/doc/help.txt

```diff
@@ -170,6 +170,8 @@ DEVELOPING NVIM
 |dev-style|		Development style guidelines
 |dev-theme|		Design guidelines (colorschemes etc.)
 |dev-tools|		Tools and techniques for developing Nvim
+|dev-vimpatch|		Merging patches from Vim
+|debug.txt|		Debugging Vim itself
```

Comment:
```suggestion
```
I removed this deliberately to promote `dev-tools` (which links to `debug.txt` at the top).